post_installation_configuration/multi-architecture-configuration.adoc

```diff
@@ -6,21 +6,17 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures. You can deploy a cluster with multi-architecture compute machines by creating an Azure installer-provisioned cluster using the multi-architecture installer binary. For Azure installation, see xref:../installing/installing_azure/installing-azure-customizations.adoc[Installing a cluster on Azure with customizations].
+An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures. Clusters with multi-architecture compute machines are only available on AWS or Azure installer-provisioned infrastructures and Bare Metal user-provisioned infrastructures with x86_64 control plane machines. 
 
-[WARNING]
-====
-The multi-architecture compute machines Technology Preview feature has limited usability with installing, upgrading, and running payloads. 
-====
+== Creating a cluster with multi-architecture compute machine on Azure 
 
-The following procedures explain how to generate an `arm64` boot image and create an Azure compute machine set with the `arm64` boot image. This adds `arm64` compute nodes to your cluster and deploys the desired amount of `arm64` virtual machines (VM). This section also shows how to upgrade your existing cluster to a cluster that supports multi-architecture compute machines. Clusters with multi-architecture compute machines are only available on Azure installer-provisioned infrastructures with `x86_64` control plane machines. 
+To deploy a Azure cluster with multi-architecture compute machines, you must first create a single-architecture Azure installer-provisioned cluster with the multi-architecture installer binary. For more information on Azure installations, refer to xref:../installing/installing_azure/installing-azure-customizations.adoc[Installing a cluster on Azure with customizations]. You can then add a ARM64 compute machine set to your cluster to create a cluster with multi-architecture compute machines. 
```

Comment:
```suggestion
To deploy an Azure cluster with multi-architecture compute machines, you must first create a single-architecture Azure installer-provisioned cluster by using the multi-architecture installer binary. For more information on Azure installations, see xref:../installing/installing_azure/installing-azure-customizations.adoc[Installing a cluster on Azure with customizations]. You can then add an ARM64 compute machine set to your cluster to create a cluster with multi-architecture compute machines. 
```

Reply:
The SSG advises using "see" instead of "refer to".

Reply:
Did the formatting change for `arm64` it looks like it is ARM64 here. Is that correct or should it stay `arm64`? Based on everywhere else here, it looks like it needs to be `arm64`

Reply:
I'm currently doing an update on all the docs for changing the architecture format based on the new guidelines. I'll update that in the 4.13 PR for the term updates.